src/js/ripple/log.js

```diff
@@ -149,7 +149,11 @@ var NullLogEngine = {
 if (typeof window !== 'undefined' && typeof console !== 'undefined') {
   Log.engine = InteractiveLogEngine;
 } else if (typeof console !== 'undefined' && console.log) {
-  Log.engine = BasicLogEngine;
+  if (process.env.NODE_ENV === 'test') {
+    Log.engine = NullLogEngine;
```

Comment:
Tests should cover error conditions, and ripple-lib shouldn't log with `log.js` during tests